bin/check-exercises.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+RED=$(tput setaf 1)
+GREEN=$(tput setaf 2)
+
+cd exercises
+mkdir tmp-exercises
+
+cnt=0
+for exercise in practice/*/ concept/*/; do
```

Comment:
This combining of these two directories won't work if you add concept exercises later on, as concept exercise should use an "exemplar" file, whereas, practice exercises should use an "example" file

Reply:
true, I was also thinking that the 

`cat .meta/example.chpl` would be problematic for concepts, since they use exemplar. A couple of thoughts I had

1. use `cat .meta/*.chpl > src/*.chpl`, this would work, but assumes there's only a `.chpl` file in each directory, which I guess it's fine.
2. iiuc, the example and exemplar files can be named arbitrary, as long as the name matches what's in the `config.json` file? One could simply document that both have to be called `example.chpl`
3. I wonder if I could achieve that with a regex, example and exemplar are not that distant.

Reply:
2b. or require just that both example and exemplar file start with a fixed prefix e.g. `ref*.chpl`, might be less confusing than requiring exemplar to be called example

Reply:
Yeah, the names can be arbitrary. I think I would tend towards something that is explicitly neither `example` nor `exemplar` if you're going to have the same filename for both. (sample-solution? reference-solution? proof?)

Reply:
makes sense, it could be

```bash
cat .meta/reference*.chpl > src/*.chpl
```

and then document in the `CONTRIBUTING.md` that both exemplar and example should start with `reference`, and everyone is free to use a more descriptive name if they want.

Or it could simply be `cat .meta/*.chpl > src/*.chpl` and not enforce naming guidelines.

Not sure which one is more robust on the longer run 🤔 , but I guess the workflow can always be adapted if/when problems arise.

Reply:
I would suggest just mandating what the name should be rather than mandating a prefix. As far as I know all the existing tracks just pick a name and stick with it.

> the workflow can always be adapted

Agreed. I wouldn't worry too much about it just now.